src/test/scala/mesosphere/marathon/api/v2/json/FormatsTest.scala

```diff
@@ -0,0 +1,110 @@
+package mesosphere.marathon.api.v2.json
+
+import java.util.UUID
+
+import mesosphere.marathon.MarathonSpec
+import mesosphere.marathon.api.v2.GroupUpdate
+import mesosphere.marathon.state.{ Timestamp, AppDefinition, Group }
+import play.api.libs.json.{ JsNull, JsValue, Format, Json }
+import mesosphere.marathon.state.PathId._
+import org.scalatest.Matchers._
+
+class FormatsTest extends MarathonSpec {
+  import Formats._
+
+  test("Can read GroupUpdate json") {
```

Comment:
Old habit: If I test functionality in class XXX, than I use the name XXXTest. That is also aligned in most IDE's (IntelliJ: Cmd+Shift T).
I am aware of the fact, that the tests for all formats is not exhaustive...